setup.cfg

```diff
@@ -1,55 +1,3 @@
-[metadata]
-name = Django
-version = attr: django.__version__
-url = https://www.djangoproject.com/
-author = Django Software Foundation
-author_email = foundation@djangoproject.com
-description = A high-level Python web framework that encourages rapid development and clean, pragmatic design.
-long_description = file: README.rst
-license = BSD-3-Clause
-classifiers =
-    Development Status :: 2 - Pre-Alpha
-    Environment :: Web Environment
-    Framework :: Django
-    Intended Audience :: Developers
-    License :: OSI Approved :: BSD License
-    Operating System :: OS Independent
-    Programming Language :: Python
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.10
-    Programming Language :: Python :: 3.11
-    Programming Language :: Python :: 3.12
-    Topic :: Internet :: WWW/HTTP
-    Topic :: Internet :: WWW/HTTP :: Dynamic Content
-    Topic :: Internet :: WWW/HTTP :: WSGI
-    Topic :: Software Development :: Libraries :: Application Frameworks
-    Topic :: Software Development :: Libraries :: Python Modules
-project_urls =
-    Documentation = https://docs.djangoproject.com/
-    Release notes = https://docs.djangoproject.com/en/stable/releases/
-    Funding = https://www.djangoproject.com/fundraising/
-    Source = https://github.com/django/django
-    Tracker = https://code.djangoproject.com/
-
-[options]
-python_requires = >=3.10
-packages = find:
-include_package_data = true
-zip_safe = false
```

Comment:
I checked the contents of the sdist archives before and after and we are missing a single file with these changes: `Django.egg-info/`[`not-zip-safe`](https://setuptools.pypa.io/en/latest/deprecated/python_eggs.html#zip-safe-and-not-zip-safe). This is missing because of not migrating over the obsolete [`zip-safe`](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#setuptools-specific-configuration) option. This is fine 👍🏻